src/RoverPhoto/RoverPhoto.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
Massive nitpick but can you change this file to RoverPhotos? It sounds like you are only displaying one photo in this component when it's actually all of them